phoenix-pherf/src/test/java/org/apache/phoenix/pherf/ResourceTest.java

```diff
@@ -51,6 +51,7 @@ public void testResourceListPropertyDirectory() throws Exception {
         PherfConstants constants = PherfConstants.create();
         assertNotNull(constants.getProperty("pherf.default.dataloader.threadpool"));
         assertNotNull(constants.getProperty("pherf.default.results.dir"));
+        assertNotNull(constants.getProperty("pherf.default.log_per_nrows"));
```

Comment:
Should use constant for the string

Reply:
used constant `PherfConstants.LOG_PER_NROWS_NAME` instead of `string`